codec_test.go

```diff
@@ -747,6 +747,22 @@ var disassembleResponseStruct = DisassembleResponse{
 	},
 }
 
+// -------- StartDebugging
+
+var startDebuggingRequestString = `{"seq":39,"type":"request","command":"startDebugging","arguments":{"request":"launch","configuration":{"any":true}}}`
+var startDebuggingRequestStruct = StartDebuggingRequest{
+	Request: *newRequest(39, "StartDebugging"),
```

Comment:
`"StartDebugging"` should be `"startDebugging"`

Reply:
Done